_includes/desktop-experimental.md

```diff
@@ -0,0 +1,5 @@
+On the **Experimental features** tab, you have the option to allow feature flags, which are features that Docker is currently experimenting with. This is switched on by default.
+
+These features are intended for testing and feedback only as they may change between releases without warning or remove them entirely from a future release. Docker does not offer support for experimental features. .
+
+For a list of current experimental features in the Docker CLI, see [Docker CLI Experimental features](https://github.com/docker/cli/blob/master/experimental/README.md){:target="_blank" rel="noopener" class="_"}.
```

Comment:
These are actually separate from experimental features in the UI, so I would just remove this sentence, or say something like you can see which experimental feature flags you have access to by checking your settings.